TheMET/TheMET/ViewControllers/FavoritesViewController.swift

```diff
@@ -68,8 +68,21 @@ class FavoritesViewController: UIViewController, UITableViewDelegate, UITableVie
         self.tableView.dataSource = self
         self.tableView.delegate = self
         self.searchBar.delegate = self
+        self.hideKeyboard()
     }
    
+    private func hideKeyboard() {
+        let tap: UIGestureRecognizer  = UITapGestureRecognizer(
+            target: self,
+            action: #selector(dissmissKeyboard)
+        )
+        self.view.addGestureRecognizer(tap)
```

Comment:
This is a good solution, but it needs to be optimized. Currently you override all the taps on the view and hide keyboard when it happens. But it also means all the other taps don’t work(Image tap, heart-icon tap, etc.)